backend/migrations/20210210112014_refactor_for_lang.sql

```diff
@@ -0,0 +1,34 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE books DROP COLUMN read_video;
+ALTER TABLE books DROP COLUMN read_body;
+ALTER TABLE books DROP COLUMN explore_video;
+ALTER TABLE books DROP COLUMN explore_body;
+ALTER TABLE books DROP COLUMN learn_video;
+ALTER TABLE books DROP COLUMN learn_body;
```

Comment:
If we had staging up already I would ask you to migrate the data from books to book_contents (with language en). But you lucked out on timing, so you don't have to do that :))))